app/src/main/java/com/duckduckgo/app/launch/LaunchViewModel.kt

```diff
@@ -44,6 +46,7 @@ class LaunchViewModel @Inject constructor(
 
     suspend fun determineViewToShow() {
         waitForReferrerData()
+        tabDataRepository.setIsUserNew(userStageStore.isNewUser())
```

Comment:
sorry I’m late to the party, but this doesn’t belong here. What does the LaunchViewModel have to do with Tabs? There are better ways to make a default value based on the stage.